src/fairseq2/recipes/lm/text_generate.py

```diff
@@ -106,7 +107,7 @@ class SamplingConfig:
     min_gen_len: int = 1
     """The minimum generation length."""
 
-    max_gen_len: int = 512
+    max_gen_len: int = 2048
```

Comment:
changing this to a more reasonable length by default since 512 is too small, and sampler handles EOS nicely, so overgenerations won't be an issue